src/lib/pages/schedules.svelte

```diff
@@ -41,38 +41,38 @@
       : schedules;
 </script>
 
-<header class="flex flex-row justify-between gap-2">
-  <div>
-    <h1
-      class="flex flex-col gap-0 text-lg md:flex-row md:items-center md:gap-2 md:text-2xl"
-    >
-      {translate('common.schedules')}
-    </h1>
-  </div>
-  {#if hasSchedules && !createDisabled}
-    <Button
-      data-testid="create-schedule"
-      href={routeForScheduleCreate({ namespace })}
-    >
-      {translate('schedules.create')}
-    </Button>
-  {/if}
-</header>
-
-{#await fetchSchedules}
-  <Loading />
-{:then { schedules, error }}
-  {#if schedules?.length}
-    <Pagination
-      items={filteredSchedules(schedules)}
-      let:visibleItems
-      aria-label={translate('common.schedules')}
-      pageSizeSelectLabel={translate('common.per-page')}
-      previousButtonLabel={translate('common.previous')}
-      nextButtonLabel={translate('common.next')}
-    >
-      <svelte:fragment slot="action-top-left">
-        <div class="w-full xl:w-1/2">
+{#key namespace}
+  <ApiPagination
+    let:visibleItems
+    {onFetch}
+    {onError}
+    aria-label="schedules"
+    pageSizeSelectLabel={translate('common.per-page')}
+    nextButtonLabel={translate('common.next')}
+    previousButtonLabel={translate('common.previous')}
+    emptyStateMessage={translate('schedules.empty-state-title')}
+    fallbackErrorMessage={'Error fetching schedules'}
```

Comment:
Looks like we use "Error fetching schedules" a few places, maybe worth defining in i18n?

Reply:
👍